speech/api/Recognize/Recognize.cs

```diff
@@ -443,6 +446,29 @@ static object SyncRecognizeGcs(string storageUri)
         }
         // [END speech_transcribe_sync_gcs]
 
+        // [START speech_transcribe_with_profanity_filter_sync_gcs]
+        static object SyncRecognizeWithProfanityFilterGcs(string storageUri)
+        {
+            var speech = SpeechClient.Create();
+            var response = speech.Recognize(new RecognitionConfig()
+            {
+                Encoding = RecognitionConfig.Types.AudioEncoding.Linear16,
+                SampleRateHertz = 16000,
```

Comment:
@b-loved-dreamer  This one is throwing and below Exception while running test Please take a look:
` Error: Grpc.Core.RpcException : Status(StatusCode=InvalidArgument, Detail="sample_rate_hertz (16000) in RecognitionConfig must either be omitted or match the value in the WAV header ( 8000).") `

Reply:
@laljikanjareeya thank you for reviewing. I will take a look.